tests/e2e/scale/test_pvc_creation_deletion_scale.py

```diff
@@ -277,3 +279,14 @@ def test_all_4_type_pvc_creation_deletion_scale(self, namespace, tmp_path):
             for k, v in fs_pvc_deletion_time.items():
                 csv_obj.writerow([k, v])
         logging.info(f"Delete data present in {log_path}-deletion-time.csv file")
+
+
+def get_pvc_count():
```

Comment:
@wusui can you move this function to some generic file ? i.e. scale_lib.py would be better.
The reason is, we might need this function for pod creations as well for other tests.

Reply:
I will move this to scale_lib